monitors-driver-library/src/main/java/cloud/benchflow/monitors/Monitor.java

```diff
@@ -0,0 +1,55 @@
+package cloud.benchflow.monitors;
+
+import com.sun.faban.driver.transport.hc3.ApacheHC3Transport;
+
+import java.util.Map;
+import java.util.logging.Logger;
+
+
+/**
+ * @author Simone D'Avico (simonedavico@gmail.com)
+ *
+ * Created on 13/07/16.
+ */
+public abstract class Monitor {
+
+    protected Map<String, String> params;
+    protected String endpoint;
+    protected MonitorAPI api;
+    protected ApacheHC3Transport http;
```

Comment:
Why the Faban transport? The plain apache http client should be fine, and it is already used by Faban behind the scenes. The same applies to collectors.